doc/manual/src/contributing/testing.md

```diff
@@ -30,32 +30,69 @@ The unit tests are defined using the [googletest] and [rapidcheck] frameworks.
 > ├── libexpr
 > │   ├── value/context.hh
 > │   ├── value/context.cc
+> │   …
+> │
+> ├── tests
 > │   │
 > │   …
->     └── tests
-> │       ├── value/context.hh
-> │       ├── value/context.cc
+> │   └── unit
+> │       ├── libutil
+> │       │   │
+> │       │   …
 > │       │
+> │       ├── libexpr
+> │       │   └── test
+> │       │       ├── value/context.hh
+> │       │       ├── value/context.cc
+> │       │       …
 > │       …
-> │
-> ├── unit-test-data
-> │   ├── libstore
-> │   │   ├── worker-protocol/content-address.bin
-> │   │   …
-> │   …
+> │       └── data
+> │           ├── libutil
+> │           │   ├── git/tree.txt
+> │           │   …
+> │           ├── libexpr
+> │           │   │
+> │           │   …
+> │           …
 > …
 > ```
 
-The unit tests for each Nix library (`libnixexpr`, `libnixstore`, etc..) live inside a directory `src/${library_shortname}/tests` within the directory for the library (`src/${library_shortname}`).
+The tests for each Nix library (`libnixexpr`, `libnixstore`, etc..) live inside a directory `src/${library_name_without-nix}-tests` next to the directory for the library (`src/${library_name_without-nix}`).
```

Comment:
This is now wrong, right?

Reply:
Good catch. There are a few things that need to be fixed here.